csrankings-3.csv

```diff
@@ -1010,6 +1010,7 @@ Hong-Han Shuai,National Chiao Tung University,https://basiclab.nctu.edu.tw,NOSCH
 Hong-Sheng Zhou,Virginia Commonwealth University,http://www.people.vcu.edu/~hszhou,PQl5OHMAAAAJ
 Hong-Tao Lu,Shanghai Jiao Tong University,http://english.seiee.sjtu.edu.cn/english/detail/841_674.htm,GtNuBJcAAAAJ
 Hongbin Zha,Peking University,http://www.cis.pku.edu.cn/vision/Visual&Robot/people/zha,LQxSSgYAAAAJ
+Hongbo Fu, City University of Hong Kong,https://sweb.cityu.edu.hk/hongbofu/,S7D5ygcAAAAJ

```

Comment:
Spurious space after comma, but more crucially, not obviously in a CS department. Please indicate why this faculty member meets the inclusion criteria.

Reply:
Hi Emery,

Thanks for pointing out the space problem. I'll fix it. For the criteria, although Prof. Hongbo Fu is not affiliated with CS department, he supervises PhD students in CS track in SCM, CityU. His research fields are in Computer Graphics and Human Computer Interaction. 
Thanks for your help.

Best,
Wanchao